frontend/src/components/public_group/PublicGroupThanksV2.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
-import ShareIcon from '../ShareIcon';
 
 export default class PublicGroupThanksV2 extends React.Component {
+
+  constructor(props) {
+    super(props);
+    const { group } = this.props;
+    const mention = group.twitterHandle ? `@${group.twitterHandle}` : group.name;
+    const tweet = `🎉 I'm now a proud supporter of ${mention}. You should support them too! https://opencollective.com/${group.slug} #opencollective`;
```

Comment:
I'm not in favor of language involving verbs such as "should", "must" or "have to". These are compassion-blocking formulations. Cfr. nonviolent communication by Marshall Rosenberg.